<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankcardWindow</class>
 <widget class="QDialog" name="BankcardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>49</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Number">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>51</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Year">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Month">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Month:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Year">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>146</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2000</number>
   </property>
   <property name="maximum">
    <number>2200</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Month">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>146</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Title">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Number">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
